src/Visitor/ParentConnectingVisitor.php

```diff
@@ -16,6 +16,7 @@
 
 /**
  * @see https://github.com/nikic/PHP-Parser/blob/v4.6.0/doc/component/FAQ.markdown#how-can-the-parent-of-a-node-be-obtained
+ * @deprecated since 1.1 and will be removed in 2.0. Use PhpParser\NodeVisitor\ParentConnectingVisitor instead.
```

Comment:
Let me know if you want to raise some kind of silenced deprecation error too, perhaps in a the constructor.